### ISSUE TITLE
FormlyPrimeNGModule broken in v6.0.0-beta.x 

### ISSUE THREAD

**Issue by YaoaY** [ngx-formly/ngx-formly#3339]
**Description**
`@ngx-formly/primeng/lib/ui-primeng.module.d`:

in v6.0.0-next.9 (works):
```
import * as i0 from "@angular/core";
import * as i1 from "@ngx-formly/primeng/form-field";
import * as i2 from "@ngx-formly/primeng/input";
import * as i3 from "@ngx-formly/primeng/textarea";
import * as i4 from "@ngx-formly/primeng/radio";
import * as i5 from "@ngx-formly/primeng/checkbox";
import * as i6 from "@ngx-formly/primeng/select";
import * as i7 from "@ngx-formly/primeng/datepicker";
export declare class FormlyPrimeNGModule {
    static ɵfac: i0.ɵɵFactoryDeclaration<FormlyPrimeNGModule, never>;
    static ɵmod: i0.ɵɵNgModuleDeclaration<FormlyPrimeNGModule, never, [typeof i1.FormlyFormFieldModule, typeof i2.FormlyInputModule, typeof i3.FormlyTextAreaModule, typeof i4.FormlyRadioModule, typeof i5.FormlyCheckboxModule, typeof i6.FormlySelectModule, typeof i7.FormlyDatePickerModule], never>;
    static ɵinj: i0.ɵɵInjectorDeclaration<FormlyPrimeNGModule>;
}

```
in v6.0.0-beta.x (😭 ):
```
export declare class FormlyPrimeNGModule {}
```

**Comment by aitboudad** [comment]
Will check further, in the meantime can you make a copy of https://github.com/ngx-formly/ngx-formly/blob/main/src/ui/primeng/src/lib/ui-primeng.module.ts and check if it works

**Comment by aitboudad** [comment]
This issue has been fixed and released as part of [v6.0.0-beta.3](https://github.com/ngx-formly/ngx-formly/releases/tag/v6.0.0-beta.3) release.

Please let us know, in case you are still encountering a similar issue/problem.
Thank you!
